src/chart/hawkular-chart-directive.ts

```diff
@@ -446,11 +447,11 @@ module Charts {
           function formatBucketedChartOutput(response) {
             //  The schema is different for bucketed output
             if (response) {
-
               return response.map((point:IChartDataPoint) => {
+                var timestamp = point.start + (point.start - point.end)/2;
```

Comment:
The data being returned is for the time between the start and the end values. Which is why I think it makes sense to use the midpoint between them.

Eg say the bucket is for the timespan between 10:00 and 10:30, I would think putting the dot at 10:15 makes more sense than putting it at 10:00

But looking at the code, I made a huge mistake here. It should be:

point.start + (point.end - point.start)/2

instead of:

point.start + (point.start - point.end)/2